src/main/java/Duke.java

```diff
@@ -1,10 +1,65 @@
+import java.util.*;
```

Comment:
Instead of importing the enttire package, it would be better to just import the class you need